.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .test
-.idea
\ No newline at end of file
+.idea
+phantom.js
```

Comment:
huh? Why did you add these? They are not automatically compiled any where. So did I miss a fix some where else?